src/index.html

```diff
@@ -43,34 +43,43 @@ <h3>
         </article>
       <article class="addActivity__article">
         <form method="POST">
-          <label>Date</label>
-          <input aria-required="true" class="form-control" type="text" name="date" placeholder="YYYY/DD/MM">
-          <label>Number of Steps</label>
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="18000" placeholder="#">
-          <label>Minutes Active</label>
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
-          <label>Flights of Stairs</label>
-          <input aria-required="true" class="form-control" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
+          <label for="form__control--date1">Date
+          <input aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
+          </label>
+          <label for="form__control--steps">Number of Steps
+          <input aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
```

Comment:
Do we need to add a similar pattern attribute to our splash page input for the user to enter their id?